diff -Naur mpeghdec-r3.0.1.old/src/CMakeLists.txt mpeghdec-r3.0.1/src/CMakeLists.txt
--- mpeghdec-r3.0.1.old/src/CMakeLists.txt	2025-09-30 17:44:56.416392046 +0200
+++ mpeghdec-r3.0.1/src/CMakeLists.txt	2025-09-30 18:13:53.443230354 +0200
@@ -9,6 +9,7 @@
 
 add_library(mpeghdec)
 install(TARGETS mpeghdec)
+set_target_properties(mpeghdec PROPERTIES VERSION ${PROJECT_VERSION} SOVERSION 3)
 target_include_directories(mpeghdec PUBLIC "${PROJECT_SOURCE_DIR}/include")
 target_include_directories(mpeghdec PRIVATE "${PROJECT_SOURCE_DIR}/include/sys")
 
