diff -Naur orig/CMakeLists.txt mpeghdec-r4.0.1/CMakeLists.txt
--- a/CMakeLists.txt	2026-07-24 12:25:09.000000000 +0200
+++ b/CMakeLists.txt	2026-09-18 11:37:35.905294142 +0200
@@ -65,6 +65,7 @@
     CACHE FILEPATH "Path to the symbol-prefix header")
 
 # Add libraries
+include(GNUInstallDirs)
 add_subdirectory(src)
 
 # Add binaries
@@ -78,7 +79,6 @@
 endif()
 
 # Install
-include(GNUInstallDirs)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/mpeghdec.pc.in ${CMAKE_CURRENT_BINARY_DIR}/mpeghdec.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/mpeghdec.pc DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/pkgconfig)
 
diff -Naur orig/src/CMakeLists.txt mpeghdec-r4.0.1/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2026-07-24 12:25:09.000000000 +0200
+++ b/src/CMakeLists.txt	2026-09-18 11:37:35.905334247 +0200
@@ -9,6 +9,10 @@
 
 add_library(mpeghdec)
 install(TARGETS mpeghdec)
+set_target_properties(mpeghdec PROPERTIES
+  VERSION ${PROJECT_VERSION}
+  SOVERSION ${PROJECT_VERSION_MAJOR}
+)
 
 # ---------------------------------------------------------------------------
 # Force-include the symbol-prefix header (see mpeghdec_SYMBOL_PREFIX in the root CMakeLists.txt file).
